presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -146,6 +146,11 @@ protected boolean isColumnNameRejected(Exception exception, String columnName, b
             return Optional.empty();
         }
 
+        if (typeName.equals("boolean")) {
+            // MySql does not have built-in support for boolean type. MySQL provides BOOLEAN as the synonym of TINYINT(1)
+            return Optional.empty();
```

Comment:
This probably means we should map boolean into tinyint in mysqlclient.
Can you file an issue?

Reply:
done.